site/docs/latest/cookbooks/tiered-storage.md

```diff
@@ -5,6 +5,8 @@ tags: [admin, tiered-storage]
 
 Pulsar's **Tiered Storage** feature allows older backlog data to be offloaded to long term storage, thereby freeing up space in BookKeeper and reducing storage costs. This cookbook walks you through using tiered storage in your Pulsar cluster.
 
+Tiered storage currently leverage [Apache Jclouds](https://jclouds.apache.org) to supports [S3](https://aws.amazon.com/s3/) and [Google Cloud Storage](https://cloud.google.com/storage/)(GCS for short) for long term storage. And by Jclouds, it is easy to add more [supported](https://jclouds.apache.org/reference/providers/#blobstore-providers) cloud storage provider in the future.
```

Comment:
-> currently uses [Apache Jclouds](...) to support [Amazon S3](...).
And by Jclouds.. -> With jclouds, it is easy to add support for more cloud storage providers in the future.

jclouds always seem to write their name in all lowercase.